services/web/remix.config.js

```diff
@@ -1,8 +1,13 @@
+import { remix } from "@remix-run/dev";
+import MillionLint from "@million/lint";
+import { defineConfig } from "vite";
+
 /** @type {import('@remix-run/dev').AppConfig} */
 export default {
-  ignoredRouteFiles: ["**/.*"]
+  ignoredRouteFiles: ["**/.*"],
+  plugins: [MillionLint.vite(), remix()]
```

Comment:
Good just rename the file / merge with latest changes in main - this is now vite.config.ts